compiler/modules/nback/build.bal

```diff
@@ -379,7 +374,6 @@ function buildAssign(llvm:Builder builder, Scaffold scaffold, bir:AssignInsn ins
 function buildCall(llvm:Builder builder, Scaffold scaffold, bir:CallInsn insn) returns BuildError? {
     // Handler indirect calls later
     bir:FunctionRef funcRef = <bir:FunctionRef>insn.func;
-    // JBUG #31008 cannot write this with select
```

Comment:
But you have not rewritten it with `select`

Reply:
When the original comment was written, only `insn.args` was being iterated. Now that the `paramTypes` is being iterated along with it, I can't do it trivially.

I may if I write a `zip` iterator, is that a good option?

Reply:
Oh I see. Never mind then.